### DISCUSSION TITLE
Relationships

### DISCUSSION THREAD

**Discussion by bameyrick** [ONSdigital/design-system#1390] [Feedback]
Use this place to discuss the [relationships pattern](https://ons-design-system.netlify.com/patterns/relationships) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by bameyrick** [comment]
The JavaScript to change the question, legend, and playback needs to be run on page load for when a user returns to a the page with their option preselected.

**Reply by jrbarnes9** [reply]
#479 resolved

**Comment by jrbarnes9** [comment]
Typos in relationships pattern documentation...
> Help users select define relationships between household memebers.

**Reply by jrbarnes9** [reply]
#811 resolved

**Comment by jrbarnes9** [comment]
Relationships "first person" example is wrong. 

The heading is the same as the third person example, whereas it should be "Joe Bloggs is your ..."

**Reply by jrbarnes9** [reply]
Resolved in #1618
